pkg/nsx/services/vpc/store_test.go

```diff
@@ -0,0 +1,251 @@
+package vpc
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/agiledragon/gomonkey"
```

Comment:
![image](https://user-images.githubusercontent.com/4385934/203456079-cb5df251-c9c3-4e84-8a60-ca9c0c5771e7.png)
You can set your IDE auto-sorting import when saving the file.